libs/s25main/lua/LuaWorld.cpp

```diff
@@ -33,7 +33,8 @@ void LuaWorld::Register(kaguya::State& state)
     state["World"].setClass(kaguya::UserdataMetatable<LuaWorld>()
                               .addFunction("AddEnvObject", AddEnvObjectWrapper())
                               .addFunction("AddStaticObject", AddStaticObjectWrapper())
-                              .addFunction("AddAnimal", &LuaWorld::AddAnimal));
+                              .addFunction("AddAnimal", &LuaWorld::AddAnimal)
+                              .addFunction("SetComputerBarrier", &LuaWorld::SetComputerBarrier));
```

Comment:
This needs increasing the LUA interface version such that mission scripts can require it if using this new function

Reply:
Bumped feature level from 4 to 5. Not sure if this is what you meant.